tutorials/500_spatiotemporal.ipynb

```diff
@@ -7,23 +7,43 @@
    "source": [
```

Comment:
<ul><li>typo <code>knowledge</code>,would prefer another term though, e.g. <code>...across time incorporating spatial information</code>.</li><li>only introduce symbols/parameters which you need later explicitly (e.g. in a formula), if you don't use the C_x, C_y, C, I think it's not necessary to have them here, whereas it's fair to keep alpha and epsilon, as these are parameters which need to be explained, as you use them in the <code>solve</code> method </li><li><br></li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>okay. other tutorials shall be modified accordingly as well.</p>

Reply:
Which notebook are you talking about?

Reply:
`100_lineage.ipynb`
and
`200_temporal_problem.ipynb`